.github/workflows/selfdrive_tests.yaml

```diff
@@ -412,22 +412,22 @@ jobs:
           echo "::set-output name=diff::$output"
       - name: Find comment
         if: ${{ env.AZURE_TOKEN != '' }}
-        uses: peter-evans/find-comment@1769778a0c5bd330272d749d12c036d65e70d39d
+        uses: peter-evans/find-comment@1769778a0c5bd330272d749d12c036d65e70d39d # pin@v2.0.0
         id: fc
         with:
           issue-number: ${{ github.event.pull_request.number }}
           body-includes: This PR makes changes to
       - name: Update comment
         if: ${{ steps.save_diff.outputs.diff != '' && env.AZURE_TOKEN != '' }}
-        uses: peter-evans/create-or-update-comment@b95e16d2859ad843a14218d1028da5b2c4cbc4b4
+        uses: peter-evans/create-or-update-comment@c9fcb64660bc90ec1cc535646af190c992007c32 # pin@v2.0.0
```

Comment:
this seems like a much more likely attack: subtly changing something like this hash in what's presented as a null-effect PR

Reply:
I would have left this one the same since its already pinned but tracking down that SHA without a version comment or something of the sort was tricky. Instead I just used the latest version from `peter-evans`. I use this exact Action myself and have checked the source code as well on this version